### ISSUE TITLE
unexpected address type in verbose mode during dynamic analysis

### ISSUE THREAD

**Issue by williballenthin** [mandiant/capa#1702]
```
@williballenthin ➜ /workspaces/capa (dynamic-feature-extraction) $ python -m capa.main -r rules1 tests/data/dynamic/cape/v2.2/d46900384c78863420fb3e297d0a2f743cd2b6b3f7f82bf64059a168e07aceb7.json -v
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/workspaces/capa/capa/main.py", line 1655, in <module>
    sys.exit(main())
  File "/workspaces/capa/capa/main.py", line 1601, in main
    print(capa.render.verbose.render(meta, rules, capabilities))
  File "/workspaces/capa/capa/render/verbose.py", line 236, in render
    return render_verbose(rd.ResultDocument.from_capa(meta, rules, capabilities))
  File "/workspaces/capa/capa/render/verbose.py", line 229, in render_verbose
    render_rules(ostream, doc)
  File "/workspaces/capa/capa/render/verbose.py", line 214, in render_rules
    rows.append(("matches", "\n".join(map(format_address, locations))))
  File "/workspaces/capa/capa/render/verbose.py", line 77, in format_address
    raise ValueError("unexpected address type")
ValueError: unexpected address type
```

d46900384c78863420fb3e297d0a2f743cd2b6b3f7f82bf64059a168e07aceb7 from v2.2 archive in https://github.com/mandiant/capa/issues/1698

  - [x] add test demonstrating rendering of the currently broken address type
  - [ ] fix the issue

**Comment by williballenthin** [comment]
same error in vverbose mode

**Comment by williballenthin** [comment]
maybe missing CallAddress here: https://github.com/mandiant/capa/blob/19495f69d747bff25ae3bfe4022a6a7c88d18a68/capa/render/verbose.py#L73

**Comment by williballenthin** [comment]
closed in #1709
